README.md

```diff
@@ -64,6 +64,28 @@ Then run:
 
 So you can `ssh myserver.com:5202` to ssh to your NAS.
 
+4. If you want to run server as service you can follow the steps given below:
+
```

Comment:
I would like this section in `examples/systemd/README.md` and put a link to it here. Running as a service is a common use case but requires efforts and understanding to do it right. That means it's not easy to explain it clearly in quick start, which should be short and concise. Also some people don't need at all.